proposals/billboards-xformOp/README.md

```diff
@@ -0,0 +1,320 @@
+}
+```
+In this example both `Cones` and `Cone2` defined their local transformations. The local
+transformation defined in `Cones` is a left-multiplied matrix of the local transformation
+of `Cone2`. In other words, transformation of `Cones` is a part of the transformation stack
+of `Cone2`. If you flatten the transformation stack, `Cone2` inherently defined an ordered
+set of `XformOp`s of [*Cones*:`xformOp:translate`, *Cones*:`xformOp:rotateXYZ`, `xformOp:translate`].
+
+Currently USD only support **simple** `xformOp`s. **Simple** means a transformation matrix
+can be computed solely on Op's attributes. In our previous example, the matrix of
+`xformOp:translate` can be computed with the authored offset vector ```(1.0, 0.0, 0.0)```.
+There are no external parameters required in this matrix computation. All `XformOp`s supported
+in USD right now have and only need one attribute to compute its local transformation.
+
+Billboards transformation is not simple. It is **compound** and **extrinsic**.
+
+### Examples of billboards transformation
+![Examples of billboards transformation](billboards-examples.png)
+
+This screenshot demonstrates some use cases of the billboards transformation:
+  - Callout: display screen facing texts at a fixed screen location with a fixed screen size.
+  - Pin: display a screen facing pin with a fixed scale.
+  - Axis tripod: display an axis tripod at a fixed screen location with a fixed screen size.
+
+### Billboards Transformation
+A billboards transformation places the renderable (or transformable in USD)
+in a predefined screen location, with a constant pixel-scaled size, or a
+locked facing alignment. A billboards transformation may consist of three types
+of transformations: affix the position, affix the size and align the front facing.
+Each of these transformations is defined as a `XformOp` in USD. XformOp `affixPosition`
+constructs the transformation that positions the transformable at an authored screen location.
+XformOp `affixScale` constructs the transformation that scales the renderable to an authored
+screen size. XformOp `align` constructs the transformation that maintains the orientation of
+transformable's front facing.
+
+`affixPosition`, `affixScale` and `align` are **compound** since they all require a set of
+attributes (instead of single one) to compute the transformation matrix. All three
+XformOp are **extrinsic** as they all request the current viewing states (such as
+the view matrix and the projection matrix) to compute the local transformation of the
+billboarded primitives (again, a transformable in USD).
+
+XformOps `affixPosition` and `affixScale` can specify the target position and size
+in a selected coordinate space:
+- `ndc`: coordinate is in the normalized device space ([-1.0, 1.0]x[-1.0, 1.0])
+- `view`: coordinate is in the view space, where camera is at (0,0,0) and faces the x-y plane.
+- `screen`: coordinate is in the screen space ([0, width-1]x[0, height-1] where width and
```

Comment:
The screen space is defined by the viewport setting of the renderer.  `viewport` needs to be one of the extrinsic states provided to `screen` mode..

I kinda against to use pixel directly as the position unit considering devices with verious dpi. I considered to add 'metric' option so users can define the position in inches or cms. I decided not to add it since it can be achieved with `screen` option. 

But 'pixel' or 'inch' are clearly more intuitive to non-engineer users, so I think it is reasonable to add them.